### DISCUSSION TITLE
Error when building image

### DISCUSSION THREAD

**Discussion by freelovehere** [postalserver/postal#2964] [Help with using Postal]
i just imported the repository for postal, i have not even made any changes yet and it came up with build error
, can anyone please tell me what  to do to fix the error please? i have attached a screenshot of the error. Thank you

![error](https://github.com/postalserver/postal/assets/169105514/7cab54e8-960d-4961-b166-4fafa5ac7331)

**Comment by willpower232** [comment]
You won't be able to push to the official Postal registry, you will have to change that to push to your own.
